src/applications/lgy/coe/shared/constants.js

```diff
@@ -1,37 +1,29 @@
+export const ACTIONS = {
+  DELETE_FILE: 'DELETE_FILE',
+  DOC_DESC: 'DOC_DESC',
+  DOC_TYPE: 'DOC_TYPE',
+  FILE_DESCRIPTION_REQUIRED: 'FILE_DESCRIPTION_REQUIRED',
+  FILE_UPLOAD_FAIL: 'FILE_UPLOAD_FAIL',
+  FILE_UPLOAD_PENDING: 'FILE_UPLOAD_PENDING',
+  FILE_UPLOAD_SUCESS: 'FILE_UPLOAD_SUCCESS',
+  FORM_SUBMIT_FAIL: 'FORM_SUBMIT_FAIL',
+  FORM_SUBMIT_PENDING: 'FORM_SUBMIT_PENDING',
+  FORM_SUBMIT_SUCCESS: 'FORM_SUBMIT_SUCCESS',
+};
+
 export const CALLSTATUS = {
-  pending: 'pending',
-  success: 'success',
   failed: 'failed',
-  skip: 'skipped',
   idle: 'idle',
+  pending: 'pending',
+  skip: 'skipped',
+  success: 'success',
 };
 
-/**
- * LGY API (https://int.services.lgy.va.gov/eligibility-manager/swagger-ui/index.html?configUrl=/eligibility-manager/v3/api-docs/swagger-config)
- * `/api/eligibility/determination` schema only showing:
- * ELIGIBLE, NOT_ELIGIBLE, PENDING, UNABLE_TO_DETERMINE_AUTOMATICALLY
- */
 export const COE_ELIGIBILITY_STATUS = {
+  available: 'AVAILABLE',
+  denied: 'DENIED',
   eligible: 'ELIGIBLE',
-  ineligible: 'NOT_NELIGIBLE',
```

Comment:
You might want to scrub references from the code to this status if you are going to remove it